server/utils/utils.test.ts

```diff
@@ -127,7 +127,7 @@ describe('months or days elapsed since', () => {
   it.each([
     [null, null, null],
     ['Empty string', '', null],
-    ['Months elapsed ', DateTime.now().minus({ months: 309 }), '309 months'],
```

Comment:
I think this fix of mine has been removed if you have done a merge with main.. Perhaps you could rebase with main?

Reply:
i have merged with main, you may be seeing an old version

Reply:
So, your version of main was probably not up to date when you did the merge. Merging this would effectivly remove my fix from main. Could you do the merge again?